gdal/.gitignore

```diff
@@ -75,6 +76,8 @@ apps/test_ogrsf
 apps/dumpoverviews
 apps/gdalwarpsimple
 apps/multireadtest
+port/dllbuild.prev
+port/prev_dllbuild.bat
 swig/java/build
```

Comment:
These batch scripts are in the `.gitignore` because, while NMAKE is supposed to clean them up, I found that in a build failure case one of them was left on disk. They'll get overwritten the next time the build tries to use one, but they probably shouldn't get committed on accident.